Source/WebCore/Modules/geolocation/Geolocation.h

```diff
@@ -65,6 +65,7 @@ class Geolocation final : public ScriptWrappable, public RefCounted<Geolocation>
 
     WEBCORE_EXPORT void resetAllGeolocationPermission();
     Document* document() const { return downcast<Document>(scriptExecutionContext()); }
+    RefPtr<Document> protectedDocument() const { return document(); }
```

Comment:
These functions are useful / used when we have to call a method on it without allocating a local variable.
e.g. `protectedDocument()->foo()`.
In the case we're storing in a local variable, we don't need to use protected* functions.